mloop_config.py

```diff
@@ -1,110 +1,135 @@
 import os
 import json
 import configparser
+import tomli
 import logging
+from collections import namedtuple
 
 
-def get(config_path=None):
+MloopParam = namedtuple("MloopParam", ["name", "min", "max", "start"])
+RunmanagerGlobal = namedtuple("RunmanagerGlobal", ["name", "expr", "args"])
+
+def is_global_active(config, group, category)
+    """
+    We check to see if the requeted global has been activated or not
+    """
+
+    if group in config["MLOOP"]["groups"]:
+        if ("enable" not in config[category][group]) or config[category][group]:
+            return True
+    
+    return False
+
+def prepare_globals(global_list, params_val_dict):
+    globals_to_set = {}
+    for g in global_list:
+        target = g.name
+        args = [ params_val_dict[arg] for arg in g.args ]
+
+        assert args
+
+        if g.expr:
+            val = eval(g.expr)(*args)
+        else:
+            val = args[0]
+
+        globals_to_set[target] = val
+
+    return globals_to_set
+
+
+def get(config_paths=None):
     """Creates config file from specified file, or
     creates one locally with default values.
     """

```

Comment:
This function's behavior has changed slightly, so let's update the docstring

Reply:
Updated docstring and removed most of the new code since it involved supporting .ini files.